pkg/cover/html.go

```diff
@@ -501,6 +496,9 @@ func groupCoverByModule(datas []fileStats) map[string]map[string]string {
 		funcs := fmt.Sprintf("%v / %v / %.2f%%", coveredFuncs[m], totalFuncs[m], percentCoveredFunc[m])
 		pcsInFuncs := fmt.Sprintf("%v / %v / %.2f%%", coveredPCsInFuncs[m], pcsInFuncs[m], percentPCsInFunc[m])
 		covedFuncs := fmt.Sprintf("%v / %v / %.2f%%", coveredPCsInFuncs[m], pcsInCoveredFuncs[m], percentPCsInCoveredFunc[m])
+		if m == "" {
+			m = "vmlinux"
```

Comment:
We have `sys/targets.Taget.KernelObject` for this. The main kernel object is not called "vmlinux" for all OSes we support.